### ISSUE TITLE
List picking needs better approach

### ISSUE THREAD

**Issue by ericmtsai** [sfsinger19103/election_anomaly#46]
Need better message to user before she is asked to pick from a long list (e.g., ReportingUnit). User may not know that better option (pick by RUType) is on the way
Specify the ReportingUnit for this _datafile
Pick the ReportingUnit record from the database:
                          Name ReportingUnitType_Id \
0        North Carolina;Alamance County;ABSENTEE          15  
1       North Carolina;Alamance County;ONE STOP 1          15  
2       North Carolina;Alamance County;ONE STOP 2          15  
3       North Carolina;Alamance County;ONE STOP 3          15  
4      North Carolina;Alamance County;PROVISIONAL          15  
...                         ...          ...  
3697 North Carolina;US Congress House of Representa...           7  
3698 North Carolina;US Congress House of Representa...           7  
3699 North Carolina;US Congress House of Representa...           7  
3700        North Carolina;Judicial District 22B          12  
3701                  none or unknown          29  
   OtherReportingUnitType ReportingUnitType  
0                  precinct  
1                  precinct  
2                  precinct  
3                  precinct  
4                  precinct  
...           ...        ...  
3697              congressional  
3698              congressional  
3699              congressional  
3700                 judicial  
3701    none or unknown       other  
[3702 rows x 4 columns]
Enter the number of the desired ReportingUnit (or nothing, if your choice is not on the list):

**Comment by ericmtsai** [comment]
I don't believe this is used as part of any user flow in the DataLoader object, so it may be able to be removed from the beta release milestone. However, note that the inclusion of the `DataLoader.display_options()` function may give the user a better way to parse through a large number of options.

**Comment by sfsinger19103** [comment]
Sounds reasonable.
